packages/server/billing/helpers/generateInvoice.ts

```diff
@@ -227,42 +227,53 @@ const makeItemDict = (stripeLineItems: Stripe.invoices.IInvoiceLineItem[]) => {
       metadata,
       period: {end},
       proration,
-      quantity
+      quantity,
+      description
     } = lineItem
-    // This has apparently changed in the new API (cannot be null). we need to fix this if we upgrade to the latest stripe API
-    const description = lineItem.description as string | null
+    const lineItemQuantity = quantity ?? 0
     if (description === null && proration === false) {
       if (!nextPeriodCharges) {
         // this must be the next month's charge
         nextPeriodCharges = new NextPeriodCharges({
           amount,
-          quantity,
+          quantity: lineItemQuantity,
           nextPeriodEnd: fromEpochSeconds(end),
-          unitPrice: lineItem.plan.amount || undefined,
-          interval: lineItem.plan.interval
+          unitPrice: lineItem.plan?.amount || undefined,
+          interval: lineItem.plan?.interval || 'month'
         })
       } else {
         //merge the quantity & price line for enterprise
         nextPeriodCharges.amount = nextPeriodCharges.amount || amount
-        nextPeriodCharges.quantity = nextPeriodCharges.quantity || quantity
+        nextPeriodCharges.quantity = nextPeriodCharges.quantity || lineItemQuantity
       }
-    } else if (!metadata.type) {
-      unknownLineItems.push(lineItem)
     } else {
-      // at this point, we don't care whether it's an auto pause or manual
-      addToDict(itemDict, lineItem)
+      if (!nextPeriodCharges) {
```

Comment:
If `description` is not null on line 234, `nextPeriodCharges` will be undefined even though we guarantee that it's non-nullable on line 222. So, if it's undefined, I'm now creating a new `nextPeriodCharges`